script/test.sh

```diff
@@ -25,6 +25,26 @@ unitTests() {
     echo "TODO: this."
 }
 
+setupCopyOfRepo() {
+    DIRECTORY=".copy-of-repo"
+    oldPWD=$(pwd)
+    
+    if [ -d "$DIRECTORY" ]; then
+        echo "Deleting old copy of repo"
+        rm -rf $DIRECTORY
+    fi
+
+    mkdir $DIRECTORY
+    cd $DIRECTORY
+
+    git init
+    cp -R $oldPWD/* .
```

Comment:
This copy is expensive when `build/` and `target/` are populated. Perhaps the the copy can be limited to what Jenkins cares about: the `vars/`, `src/` and the `resources/` folders.

The `src/test/` folder should be skipped too. Perhaps you can try this:
```
mkdir $DIRECTORY
rsync -a --exclude 'test' src/ $DIRECTORY/src
rsync -a vars/ $DIRECTORY/vars
rsync -a resources/ $DIRECTORY/resources
cd $DIRECTORY
git init
...
```

Reply:
👍 Nice! Yea, lemme give that a try